input/pagecontent/aucdi.md

```diff
@@ -60,14 +55,20 @@ Column attribute descriptions are as follows:
     <td></td>
   </tr>
   <tr>
-    <td rowspan="4">Problem/Diagnosis summary</td>
+    <td colspan="2">Last updated</td>
+    <td>TBD</td>
+    <td>TBD</td>
+    <td>This data element is added to AUCDI R1; work is underway to map to AU Core.</td>
+  </tr>
+  <tr>
+    <td rowspan="5">Problem/Diagnosis summary</td>
     <td colspan="2">Problem / Diagnosis name</td>
     <td><a href="StructureDefinition-au-core-condition.html">AU Core Condition</a></td>
     <td>Condition.code</td>
     <td></td>
   </tr>
   <tr>
-    <td colspan="2">Body site</td>
+    <td colspan="2">Body site/laterality</td>
```

Comment:
Sorry - I think this was me, could you please change back to 'Body site'. Thanks.

Reply:
Latest version I have access to says Body site/laterality. This stays for now. No change until final AUCDI is published.